CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # nf-core/proteomicslfq: Changelog
 
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
 ## v1.0dev - [date]
```

Comment:
Version here seems wrong and you did not add anything in the changelog.

Since this is the first release I would describe the features of the pipeline super briefly in the 'added' section maybe?